### ISSUE TITLE
Explore watermarks using the DUNE logo

### ISSUE THREAD

**Issue by chenel** [DUNE/dune_plot_style#40]
Not sure if we're 100% sold on this idea, but it was suggested we try using the DUNE logo as part of our watermarks rather than just the text "DUNE".  AFAICT, both ROOT and matplotlib should support this (see: TImage::Open(), example [here](https://root.cern.ch/doc/master/img2pad_8C.html), and [the relevant matplolib tutorial](https://matplotlib.org/stable/tutorials/introductory/images.html)).

Probably if we go this route we need to be able to generate it in different colors, which would be something of a hassle.  Both ROOT and matplotlib have some ways of doing that (the matplotlib tutorial above exhibits some of them), but it would take some fiddling, I think.

A sketch of what this might look like from @olantwin:
![image](https://user-images.githubusercontent.com/15690871/223490074-7d87c0d9-fbfa-40df-a13f-c97f036cf17c.png)

**Comment by olantwin** [comment]
Concerning colours, black and white, both officially provided, should be sufficient.

Thanks for looking into how this could be accomplished in practice!

**Comment by chenel** [comment]
I played with trying to add the DUNE logo (as an image) as part of the watermark in ROOT for a few hours and I'm going to admit defeat.  ROOT's image manipulation is ... patchy at best, and there doesn't seem to be a reliable way of loading a vector format image as an input at all.  (There is [a ROOT tutorial](https://root.cern.ch/doc/master/psview_8C.html) that suggests it should be possible, but my attempts to load `.pdf` or `.eps` files using that technique failed with an error that was [reported in the ROOT forums](https://root-forum.cern.ch/t/adding-an-eps-image-to-canvas/39227) but seems not to have a solution.  And even if it did, it depends on having Ghostscript installed, which is not a requirement I think we want to foist onto anyone who uses the tools...)  

Even with a bitmap (I tried the `.png` version of the DUNE logo as an experiment) I wasn't able to modify the color palette, which I think is desirable (otherwise if we decide later we need to change the watermark colors somebody needs to edit the source file).  So I think I'm going to close this issue.  If someone else is interested in resurrecting it, however, they are more than welcome...
